src/Microsoft.DocAsCode.Metadata.ManagedReference/Visitors/SymbolVisitorAdapter.cs

```diff
@@ -893,6 +895,39 @@ private NamedArgumentInfo GetNamedArgumentInfo(KeyValuePair<string, TypedConstan
             };
         }
 
+        private ConfigFilterRule LoadConfigFilterRule(string filterConfigFile)
+        {
+            ConfigFilterRule defaultRule, userRule;
+
+            var assembly = this.GetType().Assembly;
+            var defaultConfigPath = $"{assembly.GetName().Name}.Filters.defaultfilterconfig.yml";
+            using (var stream = assembly.GetManifestResourceStream(defaultConfigPath))
+            using (var reader = new StreamReader(stream))
+            {
+                defaultRule = YamlUtility.Deserialize<ConfigFilterRule>(reader);
+            }
+
+            if (string.IsNullOrEmpty(filterConfigFile))
+            {
+                return defaultRule;
+            }
+            else
+            {
+                userRule = YamlUtility.Deserialize<ConfigFilterRule>(filterConfigFile);
```

Comment:
i think it is better that the exception handler code put in YamlUtility wrapper. for this case, i don't want to swallow the user error if user wrote illegal yaml config file